positional_goal_tolerance : 0.20
robot_type: "kinova"
planner_type: "holonomic"
degrees_of_freedom: 7
dim_state: 3
m_arm: 5.0
m_base: 5.0
m_rot: 0.5
k_geo_col: 0.1
k_fin_col: 0.1
l_offset: "0.5"
weight_goal0: 0.5
radius_obst: 0.2
obstacle_type: "sphere"
radius_body_base_link: 0.4
xyz_obstacle: [10., 10., 10.]
goal_indices: 
  - 0
  - 1
  - 2
root_link: "arm_base_link"
urdf_source: "/robot_description"
end_effector_link: 'arm_end_effector_link'
orientation_helper_link: "world"
end_links: 
  - "arm_end_effector_link"
num_sphere_obs : 0
num_box_obs : 0
num_plane_constraints: 0
angular_goal_tolerance: 0.1
collision_bodies:
  radius_body_base_link: 0.4
collision_links: 
  - "base_link"
velocity_scaling: 1.0
velocity_limits:
  upper:
        - 0.3
        - 0.3
        - 0.3
  lower:
        - -0.3
        - -0.3
        - -0.3
joint_limits: [[-5, 5], [-5, 5]]
